pom.xml

```diff
@@ -461,6 +461,7 @@ Import-Package: \
                 <argument>-Dplatform.compiler=${javacpp.platform.compiler}</argument>
                 <argument>org.bytedeco.javacpp.Pointer</argument>
                 <argument>org.bytedeco.javacpp.presets.javacpp</argument>
+                <argument>org.bytedeco.javacpp.chrono.*</argument>
```

Comment:
I don't think we need to add that. What happens if you don't add this line?

Reply:
The JNI for chrono classes is not generated.

Reply:
I guess these versions of the runners were dropped. I can try to upgrade.

Reply:
Ah, I didn't update the workflow here, please do